tests/performance/test_performance.py

```diff
@@ -0,0 +1,108 @@
+import json
+import logging
+from functools import partial
+from time import process_time
+
+from databricks.labs.blueprint.parallel import Threads
+from databricks.labs.lsql.backends import SqlBackend
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service import iam
+
+from databricks.labs.ucx.workspace_access.base import Permissions
+from databricks.labs.ucx.workspace_access.groups import MigrationState
+from databricks.labs.ucx.workspace_access.manager import PermissionManager
+
+logger = logging.getLogger(__name__)
+
+NB_OF_TEST_WS_OBJECTS = 1000
+
+
+def test_apply_group_permissions_experimental_performance(
```

Comment:
can we make sure we launch it only from debugger or from local or something like that? what env variable only exists on our laptops?